frontend/zulu/src/components/Login/register.component.js

```diff
@@ -95,7 +119,8 @@ export default class Register extends Component {
       AuthService.register(
         this.state.username,
         this.state.email,
-        this.state.password
+        this.state.password,
+        this.state.fullName             // transfter the fullname //
```

Comment:
Can you remove the comment? not sure if it is needed here (code is pretty readable here)